setup.py

```diff
@@ -3,7 +3,7 @@
 """The setup script."""
 import os
 import sys
-from numpy.distutils.core import Extension, setup
+from setuptools import Extension, setup
```

Comment:
Argh indeed, I forgot about the Fortran part.
Thank you for the suggested change, I can confirm it indeed worked on my side:
- fresh install of python 3.10.6 (with `virtualenv`),
- running `pip install -r requirements.txt` where `requirements.txt` is:
```
...
../../../1-CODES/16-pymsis
pyparsing==3.0.9
pyrsistent==0.18.1
...
```

Reply:
Would you mind trying to run the CI pipeline with the change you suggested please ?

Reply:
Sorry I'm not too familiar with the CI on Github, but if I can help, please tell me.

Reply:
I'm not sure if it is in the builds, or the CI... But, looking at the logs, it looks like it potentially isn't grabbing the actual MSIS source files for compilation.

https://github.com/SWxTREC/pymsis/runs/7879692025?check_suite_focus=true#step:7:44

It could be the download script, or some path issues going on in CI... I think that means it'd be just a lot of submitting jobs to CI for debugging. I'll often put in bash commands like `ls interesting/dir` to see what was in there, or other ways to try and pipe prints out to where we can see them.

I can try and look into this more later, but it may be a day or two, so if you are interested in playing around with the CI, feel free to! You should also be able to push up to branches on your fork and have CI run there if that is easier too.

Reply:
A couple days or a week is fine. For now I can live with having to install numpy manually before anything else :)

I'll have a look at the CI logs anyways, in case I can be of any help !